src/plugins/controls/public/control_group/control_group.scss

```diff
@@ -182,12 +182,7 @@ $controlMinWidth: $euiSize * 14;
   }
 
   &-isDragging {
-    .euiFormRow__labelWrapper {
-      opacity: 0;
-    }
-    .controlFrame__formControlLayout {
-      opacity: 0; // hide dragged control, while control is dragged its replaced with ControlClone component
-    }
+    opacity: 0; // hide dragged control, while control is dragged its replaced with ControlClone component
```

Comment:
I tested this on both the new control group (in the `examples` plugin) and the old one (in the `controls` plugin) - the more selector works in both cases. No need to target `euiFormRow__labelWrapper` and `controlFrame__formControlLayout` explicitely.